tasmota/support_tasmota.ino

```diff
@@ -351,7 +358,25 @@ void SetLedPowerIdx(uint32_t led, uint32_t state)
     } else {
       led_power &= (0xFF ^ mask);
     }
-    DigitalWrite(GPIO_LED1, led, bitRead(led_inverted, led) ? !state : state);
+	uint16_t led_pwm_set = 0;
+	if (bitRead(Settings.ledpwm_mask, led)) {
+		if (bitRead(led_inverted, led)) {
+		  if (state) {
```

Comment:
Use `ledGamma10()` to turn 0..255 range into 0..1023 with gamma correction applied.